src/components/suggestion/controller.suggestion.ts

```diff
@@ -55,6 +58,52 @@ const addSuggestion = async (req: Express.Request, res: Express.Response, next:
   }
 }
 
+const getAllSuggestions = async (req: Express.Request, res: Express.Response, next: Express.NextFunction) => {
+  try {
+    const { secretkey } = req.headers
```

Comment:
Should we have some kind of wrapper for authenticated requests instead of putting this logic in every controller? (for admin routes)

Reply:
Yes, I missed it, sorry for that, I will create a middleware to handle the authentication